classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -810,7 +810,7 @@ function reviewHistory($args, $request) {
 			'common.assigned' => $reviewAssignment->getDateAssigned(),
 			'common.notified' => $reviewAssignment->getDateNotified(),
 			'common.reminder' => $reviewAssignment->getDateReminded(),
-			'common.confirm' => $reviewAssignment->getDateConfirmed(),
+			!$reviewAssignment->getDeclined() ? 'common.confirm' : 'common.declined' => $reviewAssignment->getDateConfirmed(),
```

Comment:
For better readability, could you instead do:
`$reviewAssignment->getDeclined() ? 'common.declined' : 'common.confirm' => $reviewAssignment->getDateConfirmed(),`